### PR TITLE
Program Dashboard 

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html [hunk 1]
```diff
@@ -404,6 +404,14 @@ <h4 class="mb-4 text-capitalize">{{showingResult}}</h4>
               </svg></i>
             {{resourceService.frmelmnts?.lnk?.dashboard}}
           </li>
+          <li class="list-item" *ngIf='!isDesktopApp && permissionService.permissionAvailable' appPermission
+          [permission]='programDashboardRole' (click)="toggleSideMenu(false)" routerLink="program">
+          <i class="icon-svg icon-svg--sm icon-dashboard mr-8"><svg class="icon icon-svg--primary">
+              <use xlink:href="./assets/images/sprite.svg#dashboard"></use>
+            </svg></i>
+            Program Dashboard
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/core/components/main-header/main-header.component.html:412]: please do not hard code the value

Comment (rajeevsathish) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html:412]: for this and the dashboard the image is the same?

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html (hunk 1 above)
Comment (rakeshSgr) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: I have fixed hard code value, Image is same only, program-dashboard can be accessible only with role of PROGRAM_MANAGER and PROGRAM_DESIGNER

**Review by rajeevsathish — APPROVED** [review]

**Comment by rakeshSgr** [comment]
Code coverage 70% in local
image (1)

**Pull request closed without merging**